### PR TITLE
Set-CsTenantFederationConfiguration - example changes

### PR DESCRIPTION
**Pull request by pbafna03**

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsTenantFederationConfiguration.md [hunk 1]
```diff
@@ -116,16 +116,43 @@ $list.add("fabrikam.com")
 Set-CsTenantFederationConfiguration -AllowedDomainsAsAList $list
 ```
 
-Example 6 shows how you can add domains to the Allowed Domains using a List collection object.
+Example 6 shows how you can replace domains in the Allowed Domains using a List collection object.
 First, a List collection is created and domains are added to it, then, simply include the AllowedDomainsAsAList parameter and set the parameter value to the List object.
-When this command completes, the allowed domain list will be filled with those domains.
+When this command completes, the allowed domain list will be replaced with those domains.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTenantFederationConfiguration.md:121]:
```suggestion
When this command completes, the allowed domains list will be replaced with those domains.
```

File: skype/skype-ps/skype/Set-CsTenantFederationConfiguration.md [hunk 2]
```diff
@@ -116,16 +116,43 @@ $list.add("fabrikam.com")
...
+$list = New-Object Collections.Generic.List[String]
+$list.add("contoso.com")
+$list.add("fabrikam.com")
+Set-CsTenantFederationConfiguration -AllowedDomainsAsAList @{Add=$list}
+```
+
+Example 7 shows how you can add domains to the existing Allowed Domains using a List object.
+First, a List is created and domains are added to it, then use the Add method in the AllowedDomainsAsAList parameter to add the domains to the existing allowed domains list. When this command completes, the domains in the list will be added to any domains already on the AllowedDomains list.
+
+Note - This is supported from Teams PowerShell Module version 4.5.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTenantFederationConfiguration.md:135]:
```suggestion
**Note**: This is supported from Teams PowerShell Module version 4.5.0 onwards.
```

File: skype/skype-ps/skype/Set-CsTenantFederationConfiguration.md [hunk 3]
```diff
@@ -116,16 +116,43 @@ $list.add("fabrikam.com")
...
+$list = New-Object Collections.Generic.List[String]
+$list.add("contoso.com")
+$list.add("fabrikam.com")
+Set-CsTenantFederationConfiguration -AllowedDomainsAsAList @{Remove=$list}
+```
+
+Example 8 shows how you can remove domains from the existing Allowed Domains using a List object.
+First, a List is created and domains are added to it, then use the Remove method in the AllowedDomainsAsAList parameter to remove the domains from the existing allowed domains list. When this command completes, the domains in the list will be removed from the AllowedDomains list.
+
+Note - This is supported from Teams PowerShell Module version 4.5.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTenantFederationConfiguration.md:148]:
```suggestion
**Note**: This is supported from Teams PowerShell Module version 4.5.0 onwards.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @pbafna03, kindly commit the suggested changes. Many thanks.

**Comment by pbafna03** [comment]
@JohanFreelancer9 - done, thanks!

**Review by tseward — APPROVED** [review]

**Pull request merged**